packages/frontend/.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-	"extends": ["next/babel", "next/core-web-vitals"]
+	"extends": ["next"]
```

Comment:
This is supposed to be working now; https://stackoverflow.com/questions/71662525/failed-to-load-config-next-babel-to-extend-from-eslintrc-json